cpp/src/stream_compaction/distinct_helpers.hpp

```diff
@@ -23,11 +23,13 @@
 #include <rmm/device_uvector.hpp>
 #include <rmm/resource_ref.hpp>
 
+#include <cuco/static_map.cuh>
```

Comment:
Now with including the `cuco` header, this file should be renamed into `_helper.cuh`. Otherwise please move the these header and their relevant code into `_helper.cuh` and keep this file containing host-only code.